### ISSUE TITLE
Github auth widget responsiveness

### ISSUE THREAD

**Issue by NedHenry** [UCSCLibrary/BulkOps#12]
The github auth widget on the operation forms looks bad when the screen is narrow. More broadly, all forms should be edited for responsiveness & accessibility.
